### PR TITLE
Case study

### PR DESCRIPTION
**Pull request by Shivam-AfA**
**Description**

This PR adds the HPE case study.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Shivam-AfA** [comment]
@leecalcote We can add redirect to this case study from the home page using the inline quotes that we use on Meshmap and meshery pages.

**Comment by l5io** [comment]
🚀 Preview for commit 90eff9e1faaa2b9b7bc2acb224eb0c3bf0030237 at: https://644afbafa0f85114975dabb3--layer5.netlify.app

**Comment by leecalcote** [comment]
@Shivam-AfA have you thanked HPE for their feedback and incorporated their feedback?

**Comment by Shivam-AfA** [comment]
@leecalcote We haven't recieved any feedback yet on the document. They said that they will comment their suggestions in the document. I have thanked them in a way to remind them that we are waiting for their feedback. 

"Thank you Maximiliano. We'll look forward to your feedback on the document."

**Comment by l5io** [comment]
🚀 Preview for commit 1e94c9f6599a16170870a14ecbe6b6c24ce2a309 at: https://648641672c003b5344beab0d--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 4088e1a2ba55a8f55c0d1ba51b836bb9132a2231 at: https://648651b1f6529360cb469e6f--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 621487fb8ceaf824f9435bdba1dce24ef65ec3fb at: https://6486de63c8c20b0090f1bd06--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit b6805a3e000cc58e0e96c002d62a9979055da1eb at: https://6488bbc990bf8e09a3330a16--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f6d8d968bef907d72dbdea3df10985c02aa94d9a at: https://6489af4699071315e3ad51ef--layer5.netlify.app

File: src/collections/resources/articles/hpes-adoption-of-meshery-and-meshmap/index.mdx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+import LayersIcon from "./layers.svg"
+
+<ResourceWrapper>
+
+<HPEintro>
+   HPE's adoption of Meshery was driven by the need to simplify the management and monitoring of their Kubernetes clusters. Meshery is an open source cloud native management tool that provides a self-service platform for designing, deploying, testing, and operating cloud native infrastructure wherever Kubernetes and its workloads run.
+</HPEintro>
+
+<p>
+   HPE, a leading technology company specializing in enterprise infrastructure, adopted Layer5 MeshMap to enhance their Kubernetes deployments. HPE uses Kubernetes as a primary platform to build and deploy their containerized applications. The company has a large and complex Kubernetes environment, which requires robust networking solutions for efficient communication between services.
```
Comment (Chadha93) [inline src/collections/resources/articles/hpes-adoption-of-meshery-and-meshmap/index.mdx:36]:
```suggestion
   HPE, a leading technology company specializing in enterprise infrastructure, adopted Meshery Extension to enhance their Kubernetes deployments. HPE uses Kubernetes as a primary platform to build and deploy their containerized applications. The company has a large and complex Kubernetes environment, which requires robust networking solutions for efficient communication between services.
```

File: src/collections/resources/articles/hpes-adoption-of-meshery-and-meshmap/index.mdx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+import Button from "../../../../reusecore/Button";
+import GlobeIcon from "./globe.svg";
+import UsersIcon from "./users.svg";
+import NetworkIcon from "./network.svg";
+import LayersIcon from "./layers.svg"
+
+<ResourceWrapper>
+
+<HPEintro>
+   HPE's adoption of Meshery was driven by the need to simplify the management and monitoring of their Kubernetes clusters. Meshery is an open source cloud native management tool that provides a self-service platform for designing, deploying, testing, and operating cloud native infrastructure wherever Kubernetes and its workloads run.
```
Comment (Chadha93) [inline src/collections/resources/articles/hpes-adoption-of-meshery-and-meshmap/index.mdx:32]:
```suggestion
HPE's adoption of Meshery was driven by the need to simplify Kubernetes cluster management and monitoring. Meshery is an open source cloud native management tool that provides a self-service platform for designing, visualizing, deploying, testing, and operating cloud native infrastructure.
```

File: src/collections/resources/articles/hpes-adoption-of-meshery-and-meshmap/index.mdx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+      </td>
+      <td>
+      <img src={NetworkIcon} />
+      Research and Development: HPE invests significantly in research and development to drive innovation. It operates HPE Labs, which focuses on developing cutting-edge technologies and solutions for the future.
+      </td>
+   </tr>
+</HPEfacts>
+
+<p>
+SPIRE is a toolchain of APIs for establishing trust between software systems across a wide variety of hosting platforms. SPIRE exposes the SPIFFE Workload API, which can attest running software systems and issue SPIFFE IDs and SVIDs to them. <br/><br/>
```
Comment (Chadha93) [inline src/collections/resources/articles/hpes-adoption-of-meshery-and-meshmap/index.mdx:66]:
```suggestion
SPIRE is a toolchain of APIs for establishing trust between software systems across a wide variety of hosting platforms. SPIRE exposes the SPIFFE Workload API, which attests to running software systems and issues SPIFFE IDs and SVIDs to them.<br/><br/>
```

File: src/collections/resources/articles/hpes-adoption-of-meshery-and-meshmap/index.mdx [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+    <li>
+    Simplified testing and validation: Meshery's service mesh validation capabilities enable HPE to easily test and validate their service meshes, ensuring that they meet their performance, security, and compliance requirements.
+    </li>
+    <li>
+    Enhanced security: With Meshery's security features, HPE can ensure that their service meshes are secure and compliant with their organization's security policies.
+    </li>
+</p>
+
+<p>
+    Overall, Meshery has helped HPE to streamline their integration of the identity management control plane to reduce complexity, and improve the overall reliability and performance of their Kubernetes environment. SPIFFE is a set of open-source specifications for a framework capable of bootstrapping and issuing identity to services across heterogeneous environments and organizational boundaries. The lifecycle of SPIFFE identities, SVIDs, is managed by SPIRE, a production-ready implementation of the SPIFFE APIs that performs node and workload attestation in order to securely issue SVIDs to workloads, and verify the SVIDs of other workloads, based on a predefined set of conditions.
```
Comment (Chadha93) [inline src/collections/resources/articles/hpes-adoption-of-meshery-and-meshmap/index.mdx:83]:
```suggestion
Overall, Meshery has helped HPE to streamline their integration of the identity management control plane to reduce complexity, and improve the overall reliability and performance of their Kubernetes environment. SPIFFE is a set of open-source specifications for a framework capable of bootstrapping and identifying services across heterogeneous environments and organizational boundaries. The lifecycle of SPIFFE identities, SVIDs, is managed by SPIRE, a production-ready implementation of the SPIFFE APIs that performs node and workload attestation to securely issue SVIDs to workloads, and verify the SVIDs of other workloads, based on a predefined set of conditions.
```

File: src/collections/resources/articles/hpes-adoption-of-meshery-and-meshmap/index.mdx [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+</p>
+
+<p>
+   By implementing SMP, Meshery provides a valuable tool for developers and operators who are evaluating different service meshes. Instead of having to create their own benchmarks, they can use SMP to get an objective and standardized view of each service mesh's performance characteristics. This can save a significant amount of time and effort, and help users make more informed decisions when choosing a service mesh.
+</p>
+
+<p>
+Overall, HPE's use of Meshery and the Docker Extension for Meshery demonstrates the power of cloud native technologies and the importance of open source collaboration. By leveraging these tools, HPE has been able to streamline its development and deployment processes, improve performance and security, and stay at the forefront of the cloud native movement.
```
Comment (Chadha93) [inline src/collections/resources/articles/hpes-adoption-of-meshery-and-meshmap/index.mdx:127]: Let's add some quotes from HPE folks here if we could.

Comment (Shivam-AfA) [reply src/collections/resources/articles/hpes-adoption-of-meshery-and-meshmap/index.mdx:127]: @Chadha93 @leecalcote we can use the quote from Maximiliano here.

File: src/collections/resources/articles/hpes-adoption-of-meshery-and-meshmap/index.mdx (hunk 3 above)
Comment (leecalcote) [reply src/collections/resources/articles/hpes-adoption-of-meshery-and-meshmap/index.mdx:66]: The current grammar is correct.

File: src/collections/resources/articles/hpes-adoption-of-meshery-and-meshmap/index.mdx (hunk 5 above)
Comment (leecalcote) [reply src/collections/resources/articles/hpes-adoption-of-meshery-and-meshmap/index.mdx:127]: Sounds great. I'll send another from another HPE employeee.

**Comment by l5io** [comment]
🚀 Preview for commit 48453f397cc0289d09be73523245f0a3eb603bf4 at: https://648cb24cc41d5105c259bb33--layer5.netlify.app

**Review by Chadha93 — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 39fca538a4bff4e821483db5a80f39950fb63a60 at: https://648cbc9a781df8096a25f482--layer5.netlify.app

**Pull request merged**